### PR TITLE
Encode the prefix into the srw macro

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: lib/macros/srw.rb [hunk 1]
```diff
@@ -9,11 +9,14 @@ module SRW
       srw: 'http://www.loc.gov/zing/srw/'
     }.freeze
 
+    PREFIX = 'srw:record/srw:recordData/oai_dc:dc/'.freeze
```
Comment (justinlittman) [inline lib/macros/srw.rb]: Is freeze necessary? Aren't strings frozen by default?

File: lib/macros/srw.rb [hunk 2]
```diff
@@ -9,11 +9,14 @@ module SRW
...
+    private_constant :PREFIX
+
     # Extracts values for the given xpath
     # @param [String] xpath the xpath query expression
     # @return [Proc] a proc that traject can call for each record
     def extract_srw(xpath, options = {})
-      extract_xml(xpath, NS, options)
+      extract_xml("#{PREFIX}#{xpath}", NS, options)
```
Comment (justinlittman) [inline lib/macros/srw.rb]: That this is prefixed should be noted in doc comments.

File: lib/macros/srw.rb (hunk 1 above)
Comment (jcoyne) [reply lib/macros/srw.rb]: I thought so, but the version of rubocop wanted it this way. *shrug*

Comment (jcoyne) [reply lib/macros/srw.rb]: See #97

File: lib/macros/srw.rb (hunk 2 above)
Comment (jcoyne) [reply lib/macros/srw.rb]: 👍

**Pull request merged**